pkg/util/clusterdata/clusterdata.go

```diff
@@ -99,7 +107,14 @@ func (e *bestEffortEnricher) enrichOne(ctx context.Context, oc *api.OpenShiftClu
 	callbacks := make(chan func(), len(tasks))
 	errors := make(chan error, len(tasks))
 	for i := range tasks {
-		go tasks[i].FetchData(callbacks, errors)
+		go func(i int) {
+			t := time.Now()
+			defer func() {
+				e.m.EmitGauge("enricher.tasks.duration", time.Now().Sub(t).Milliseconds(), nil)
```

Comment:
Tempted to split this by task name and leave the others - wdyt?
Rationale - splitting errors by task is hard because of `e.m.EmitGauge("enricher.tasks.errors", int64(len(e.taskConstructors)), nil)`, but with logging the metric is still useful, whereas enricher.tasks.duration is of minimal use without being split out.

Reply:
@m1kola bump